roles.md

```diff
@@ -2,8 +2,125 @@
+
+## Community
+
+Everyone contributing to, or using artifacts from, The Good Docs Project is considered part of our community. We encourage all community members to actively participate in our activities. This includes:
+
+*   Participate in our forums and vote on motions.
+*   Raise and comment on bugs.
+*   Suggest fixes by raising pull requests.
+
+**Access:**
+
+*   Community members are granted _read_ access to our github repositories.
+
+**Expectations of community members:**
+
+*   Community members are expected to be respectful to each other, in line with our [Code of Conduct](https://github.com/thegooddocsproject/governance/blob/master/CodeOfConduct.md).
+
+## Committers
+
+A committer is a trusted community member who has additionally been given _write_ access to some or all of our github repositories.
+
+**Access:**
+
+*   Committers are granted _triage_, _maintain_ and _write_ access to the github repositories which they have contributed to.
+
+**Additional expectations of committers:**
+
+*   Understand and apply our quality guidelines.
+*   Understand and abide by our license requirements.
+*   Periodically contribute to our project over a one year period.
+
+**Eligibility:**
+
+Before being accepted for this role, a person should display the following characteristics:
+
+*   Provide sustained, valuable contributions to the project, typically for three months or more. An example demonstration of this could be:
+    *   Submit pull requests which have been accepted and merged by other committers. Typically one large, three medium, or five small pull requests over three months.
+    *   Or provide equivalent contributions to our community, such as outreach, contributing to our forums, or coordinating activities. 
+*   Express an interest in ongoing participation.
+*   Acknowledge the committer expectations.
+
+## Project Steering Committee
+
+The Project Steering Committee (PSC) is the official managing body for our project. The committee is drawn from the community and is based on merit and interest.
+
+*   We aim to attract a diverse mix of personal characteristics, reflective of the community we represent.
+*   We wish to avoid having one organisation employing a controlling proportion of the PSC.
+
+A PSC chair shall be selected from the PSC, with the only official extra duty being that they have a deciding vote in the case of a tie.
+
+**Access:**
+
+*   PSC members may be granted admin access to our github repositories if they have a need for them.
+
+**Additional expectations of PSC members:**
+
+*   Consider opinions from within our community, and advocate for the best interests of our community when making decisions.
+*   Vote according to your personal best judgment, as opposed to voting on behalf of your employer.
+*   Vote promptly (usually within days) to motions being raised.
+*   Regularly participate in project activities.
+*   Help with whatever unglamourous work that is required to keep the project moving forward.
```

Comment:
Personally, I like the term "plumbing" to describe this kind of work. I'm not sure any of what we do is particularly glamorous ;) (Also: needs to be corrected to US spelling)

Reply:
Fine with this being here for now, but feel like we should have a larger `Core Values` statement documented which would be inclusive of this idea.  And those that don't embody those values probably shouldn't be members of the PSC.